source/Octopus.Tentacle/Services/Scripts/KubernetesScriptServiceV1Alpha.cs

```diff
@@ -71,33 +70,21 @@ public async Task<ScriptStatusResponseV3Alpha> StartScriptAsync(StartScriptComma
                     runningScript.ScriptStateStore.Create();
                 }
 
-                if(!scriptExecutor.CanExecute(command))
-                    throw new InvalidOperationException($"The execution context type {command.ExecutionContext.GetType().Name} cannot be used with script executor {scriptExecutor.GetType().Name}.");
-
-                var process = scriptExecutor.ExecuteOnBackgroundThread(command, workspace, runningScript.ScriptStateStore, runningScript.CancellationToken);
+                var process = executor.ExecuteOnBackgroundThread(command, workspace, runningScript.ScriptStateStore, runningScript.CancellationToken);
 
                 runningScript.Process = process;
 
-                if (command.DurationToWaitForScriptToFinish != null)
-                {
-                    var waited = Stopwatch.StartNew();
-                    while (process.State != ProcessState.Complete && waited.Elapsed < command.DurationToWaitForScriptToFinish.Value)
-                    {
-                        Thread.Sleep(TimeSpan.FromMilliseconds(10));
-                    }
-                }
```

Comment:
This property is not on the command anymore and we don't want this logic as we _only_ run in polling mode